frontend/src/modules/app/layouts/ScreensaverRoute.tsx

```diff
@@ -1,7 +1,6 @@
 /**
- * This is the doc comment for Screensaver Layout
- * @packageDocumentation
- * @module screensaver-layout
+ * @summary Layout for screensaver page
```

Comment:
Where should we go to specify the content for the screensaver page?

Reply:
Main content for all the Route(layout) files are provided in Routes.tsx file. All the layout files are defined under `app/layout`